charts/clusterapi-resources/templates/ConfigMapAddClusterToArgo.yaml

```
@@ -0,0 +1,96 @@
+{{- if .Values.argo.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: add-cluster-to-argo
+  namespace: {{ .Release.Namespace }}
+data:
+  data: |-
+    ---
+    apiVersion: v1
+    kind: Pod
+    metadata:
+      name: initial-cluster-setup
+      namespace: argo
+      labels:
+        app: alpine
+    spec:
+      restartPolicy: OnFailure
+      containers:
+      - name: alpine
+        image: alpine:latest
```

Could you specify the tag? It is normally a good practice to evade unplanned changes
```suggestion
        image: alpine:latest
```